gtfs-realtime/CHANGES.md

```diff
@@ -0,0 +1,80 @@
+The GTFS-realtime specification is not set in stone. Instead, it is an open specification developed and maintained by the community of transit agencies, developers, and other stakeholders who use GTFS-realtime. It is expected that this community of producers and consumers of GTFS-realtime data will have proposals for extending the spec to enable new capabilities. To help manage that process, the following procedures and guidelines have been established.
+
+### Specification amendment process
+The official specification, reference and documentation are written in English. If a translation to a different language differs from the English original, the latter takes precedence. All communication is performed in English.
+
+1. Create a git branch with update of all relevant parts of protocol definition, specification and documentation files (except for translations).
+1. Create pull request on https://github.com/google/transit. Pull request must contain an extended description of the patch. The creator of the pull request becomes the _advocate_.
+1. Once pull request is registered, it must be announced by its advocate in the [GTFS Realtime mailing list](https://groups.google.com/forum/#!forum/gtfs-realtime). Once announced, the pull request is considered a proposal.
+  - Advocate is a contributor, thus they should sign [CLA](../CONTRIBUTING.md) before pull request can be accepted.
```

Comment:
Don't use CLA as an acronym here, as it has not yet been defined.

"Since the Advocate is a contributor, they must sign the [Contributor License Agreement](../CONTRIBUTING.md) before the pull request can be accepted."